tests/integration/security/authorization_test.go

```diff
@@ -972,6 +975,9 @@ func TestAuthorization_GRPC(t *testing.T) {
 	framework.NewTest(t).
 		Features("security.authorization.grpc-protocol").
 		Run(func(ctx framework.TestContext) {
+			if ctx.Clusters().IsMulticluster() {
+				ctx.Skip()
```

Comment:
why skip the test, need better comment. Are we going to enable multicluster by default? If so it means we lost this test until this is fixed.

Reply:
Not by default. Currently multicluster test job integ-security-multicluster-tests_istio is optional and not required by presubmit. We will turn it into required/default after the converting is done. I will raise a follow up PR to finish all authorization tests. This is a temporary flag indicated https://github.com/istio/istio/pull/28088/files/a9c1913a105b0308e10358baaae01723f9426eb0#diff-8f18b4c496e45a8c03665ecd21ebfb1a5b62aeeea7c64ef4711265bd5d23d2a1R40. Sure! I will add more comment into every skip in this PR, which will be most likely removed in the next PR.